src/NodaTime/Text/PeriodPattern.cs

```diff
@@ -193,6 +193,19 @@ public StringBuilder AppendFormat(Period value, StringBuilder builder)
                     builder.Append("P0D");
                     return builder;
                 }
+
+                if (value.Equals(Period.PositiveInfinity))
+                {
+                    builder.Append("infinity");
```

Comment:
@roji @jskeet 

Both Parse and ToString() will need to adapt to this change.  

It seems like there might be information in the [latest standard]((https://www.iso.org/obp/ui/en/#iso:std:iso:8601:-1:ed-1:v1:en) about how to do this but I can only access parts of it and will have to pay to access the whole document, so much for having a standard that you can only implement if you spend enough. I digress.


Do you have suggestions on how to represent (and expect) the string format?

Reply:
Not sure the infinity values should have any specific ToString/Parse behavior; these are not special values (in the PostgreSQL sense) - they're exactly the same as a Period manually constructed with MaxValue components...

Reply:
I see what you mean. It makes more sense and does not pollute NodaTime with Npgsql domain specifics.